noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -2245,23 +2412,21 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
-        <title>Number of screened registry entries</title>
-        <group>loependejournal</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT COUNT(tilgangsrestriksjon) screened_registry_entries
-                    FROM loependejournal.journalpost;
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    <!--
+    ============================================================================================================
+    OFFENTLIGJOURNAL
+    ============================================================================================================
+    -->
+    <!-- Tests -->
 
-    <rule>
+    <test id="OJT1">
         <title>Registry Entry count</title>
+        <description>
+            Provides information about the number of Registry Entries found in offentligjournal
```

Comment:
`Provides information about the number of registry entries found in offentligJournal.xml and tests whether it matches the number of registry entries found in arkivstruktur.xml`